src/main/resources/static/css/cas.css

```diff
@@ -919,6 +919,7 @@ body {
     padding: 1rem 0;
 }
 
+.login-instn-card .card-message,
```

Comment:
I removed all legacy oldCAS styles, the only change here is to reduce the space between vertical elements. This is used for all instn login pages and authn error pages.